tests/Cache/FileCacheTest.php

```diff
@@ -73,7 +73,7 @@ public function testBrokenCacheRead(): void
             'broken_feature',
             new FeatureNode(null, null, [], null, [], '', '', null, 1),
         );
-        $files = glob($this->path . '/**/*.feature.cache');
+        $files = glob($this->path . '/**/*.feature.cache') ?: [];
```

Comment:
Might be easier to debug & more consistent with the `file_get_contents` fixes to extract the glob calls (here & ParserTest) to a method that throws if the glob returns false?